easybuild/easyconfigs/s/simint/simint-0.7-GCCcore-11.2.0-lmax-5-vec-avx-psi4.eb

```diff
@@ -0,0 +1,54 @@
+# For the various build options see here:
+# https://www.bennyp.org/research/simint/README.txt
+# lmax is set to 5 which is widely used it seems
+# vec is set to avx, which should work on most platforms
+# This might be changed to other values, if required
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'simint'
+version = '0.7'
+local_lmax = '5'
+# supported: {scalar, sse, avx, avxfma, micavx512}
+# scalar ONLY FOR USE WITH simint-scalar (and only useful for benchmarks), micavx512 experimental!
+local_vec = 'avx'
+
+# custom configuration, to be used as dependency for PSI4
+versionsuffix = '-lmax-%s-vec-%s-psi4' % (local_lmax, local_vec)
+
+homepage = 'https://www.bennyp.org/research/simint/'
+description = """Simint is a vectorized implementation of the Obara-Saika (OS) 
+method of calculating electron repulsion integrals. Speedup is gained by 
+vectorizing the primitive loop of the OS algorithm, with additional vectorization 
+and optimizations left to the compiler."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
```

Comment:
@sassy-crick This easyconfig should have been removed, please make a follow-up PR for that?